src/nplinker/genomics/mibig/mibig_loader.py

```diff
@@ -108,6 +110,7 @@ def parse_bgc_metadata_json(file: str) -> BGC:
     metadata = MibigMetadata(file)
     mibig_bgc = BGC(metadata.mibig_accession, metadata.biosyn_class)
     mibig_bgc.mibig_bgc_class = metadata.biosyn_class
+    mibig_bgc.strain = Strain(metadata.mibig_accession)
```

Comment:
So the mibig accession is equal to the strain? What info does the mibig accession normally contain?

Reply:
No, mibig accession (a string identifier) is not equal to strain, but nplinker uses it as strain. Actually just as what I said this Monday, nplinker does not care about the real name of strain but only need a unique name or label for that. mibig accession is unique, so it's used as strain name. 
Let's keep it this way until we refactor the strain part.